Packs/Core/TestPlaybooks/Test_Playbook_-_IOC_Alert.yml

```diff
@@ -0,0 +1,648 @@
+      version: -1
+      name: Verify Extracted Indicators
+      description: Verify that the ‘ExtractedIndicators’ context key was extracted correctly.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "71"
+      "yes":
+      - "306"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: ExtractedIndicators
+            iscontext: true
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 850,
+          "y": 1200
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "71":
+    id: "71"
+    taskid: cee5c66c-f442-4deb-8200-de2684e4d81e
+    type: regular
+    task:
+      id: cee5c66c-f442-4deb-8200-de2684e4d81e
+      version: -1
+      name: 5Verify Context Error - Extracted Indicators
+      description: Prints an error entry with a given message
+      scriptName: PrintErrorEntry
+      type: regular
+      iscommand: false
+      brand: ""
+    scriptarguments:
+      message:
+        simple: |-
+          The 'ExtractedIndicators' context key was not extracted properly. One of the following may be the cause:
+          1- The 'extractIndicators' script failed to execute in the 'Extract IOC' task.
+          2- The 'text' input configuration was changed for the 'extractIndicators' automation used in the 'Extract IOC' task.
+          3- The 'extractIndicators' automation outputs have been modified and no longer contain the 'ExtractedIndicators' context key.
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 850,
+          "y": 1445
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "269":
+    id: "269"
+    taskid: 460da925-cd41-41d0-861d-a060b36133cb
+    type: title
+    task:
+      id: 460da925-cd41-41d0-861d-a060b36133cb
+      version: -1
+      name: Check Alert Field
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "270"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 1070
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "270":
+    id: "270"
+    taskid: 12e4d4d1-312c-4286-8bd3-94cd6a1c7b98
+    type: condition
+    task:
+      id: 12e4d4d1-312c-4286-8bd3-94cd6a1c7b98
+      version: -1
+      name: Verify Close Reason
```

Comment:
is this task is being checked at all? Asking since we closed the investigation in the IOC Alert ...

Reply:
We are closing the investigation in 'IOC Alert' with the 'Resolved - Threat Handled' close reason.
![Screenshot 2024-06-26 at 13 02 35](https://github.com/demisto/content/assets/112805149/780dc3df-e182-4693-a233-d68286f3f8c8)